### PR TITLE
Update documentation of gif models

### PR DESCRIPTION
**Pull request by sarakonradi**
This PR updates the links, references and equations of the following files: 
- ``gif_cond_exp.h``
- ``gif_cond_exp_multusynapse.h``
- ``gif_pop_psc_exp.h``
- ``gif_psc_exp.h``
- ``gif_psc_exp_multisynapse.h``

### CODE REVIEW

File: models/gif_pop_psc_exp.h [hunk 1]
```diff
@@ -52,15 +52,15 @@ Description
...
 multi-timescale adaptation and exponential postsynaptic currents, as
-described in [1]_.
+described in Schwalger et al. (2017) [1]_.
 
-The single neuron model is defined by the hazard function
+The single neuron model is defined by the hazard function:
 
 .. math::
 
  lambda_0 * exp[ ( V_m - E_sfa ) / Delta_V ]
```
Comment (clinssen) [inline models/gif_pop_psc_exp.h:61]:
```suggestion
 \lambda_0 * \exp\left(( V_m - E_{sfa} ) / \Delta_V\right)
```

File: models/gif_psc_exp_multisynapse.h [hunk 2]
```diff
@@ -86,28 +86,28 @@ where :math:`V_T(t)` is a time-dependent firing threshold:
 
  V_T(t) = V_{T_star} + \gamma_1(t) + \gamma_2(t) + \ldots + \gamma_m(t)
```
Comment (clinssen) [inline models/gif_psc_exp_multisynapse.h:87]:
```suggestion
 V_T(t) = V_{T_{star}} + \gamma_1(t) + \gamma_2(t) + \ldots + \gamma_m(t)
```

File: models/gif_pop_psc_exp.h (hunk 1 above)
Comment (terhorstd) [reply models/gif_pop_psc_exp.h:61]: Good point, @clinssen, we could maybe think about a general rule which we can apply to all models. This one I would leave as it is now and fix it if necessary after the discussion.

**Review by terhorstd — APPROVED** [review]
LGTM :+1:

File: models/gif_pop_psc_exp.h (hunk 1 above)
Comment (clinssen) [reply models/gif_pop_psc_exp.h:61]: We cannot leave it as-is, as it wouldn't render right; would have to escape the underscores in that case and wrap variable names in \texttt{}.

**Review by clinssen — APPROVED** [review]
Thanks for the fast updates!

**Pull request merged**